SteamcLog/Classes/SteamcLog.swift

```diff
@@ -262,6 +262,17 @@ public struct SteamcLog {
     }
 
     public func error<T>(_ message: StaticString, _ object: T, functionName: StaticString = #function, fileName: StaticString = #file, lineNumber: Int = #line) where T: Encodable {
+
+        if let error = object as? Error, config.filterOut(error) {
+            info("\(error) on blocked list and has been blocked from being captured as an error: \(error.localizedDescription)",
+                 functionName: functionName,
+                 fileName: fileName,
+                 lineNumber: lineNumber
+            )
+            return
```

Comment:
It could be handy to add a note at debug or verbose here mentioning which rule triggered the fitler

Reply:
Not sure what you mean by this? We've got the `info` call specifying that the error was filtered out via the SentryFilter (tweaked the wording above)

Reply:
I had assumed if we were able to set multiple filters, it would be helpful to log which filter caught the message